docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+   +--------------------------------+--------------------------------------------------------+
+   | Externals.cfg                  | Includes tags pointing to the correct version of the   |
+   |                                | external GitHub repositories/branches used in the SRW  |
+   |                                | App.                                                   |
+   +--------------------------------+--------------------------------------------------------+
+   | LICENSE.md                     | CC0 license information                                |
+   +--------------------------------+--------------------------------------------------------+
+   | README.md                      | Getting Started Guide                                  |
+   +--------------------------------+--------------------------------------------------------+
+   | ufs_srweather_app_meta.h.in    | Meta information for SRW App which can be used by      |
+   |                                | other packages                                         |
+   +--------------------------------+--------------------------------------------------------+
+   | ufs_srweather_app.settings.in  | SRW App configuration summary                          |
+   +--------------------------------+--------------------------------------------------------+
+   | env                            | Contains build and workflow environment files          |
+   +--------------------------------+--------------------------------------------------------+
+   | docs                           | Contains release notes, documentation, and Users' Guide|
+   +--------------------------------+--------------------------------------------------------+
+   | manage_externals               | Utility for checking out external repositories         |
+   +--------------------------------+--------------------------------------------------------+
+   | src                            | Contains CMakeLists.txt; external repositories         |
+   |                                | will be cloned in this directory.                      |
+   +--------------------------------+--------------------------------------------------------+
+
+
+.. _CheckoutExternals:
+
+Check Out External Components
+================================
+
+The SRW App relies on a variety of components (e.g., regional_workflow, UFS_UTILS, ufs-weather-model, and UPP) detailed in :numref:`Chapter %s <Components>` of this User's Guide. Users must run the ``checkout_externals`` script to link the necessary external repositories to the SRW App. The ``checkout_externals`` script uses the configuration file ``Externals.cfg`` in the top level directory of the SRW App to clone the correct tags (code versions) of the external repositories listed in :numref:`Section %s <HierarchicalRepoStr>` into the appropriate directories under the ``regional_workflow`` and ``src`` directories. 
```

Comment:
To clarify, does the SRW rely on only the regional_workflow, UFS_UTILS, ufs-weather-model, UPP (and potentially METplus) components, or are those just the most notable? I hesitate to mention METplus precisely because it's optional, and I think the goal is to make these instructions as generally-applicable as possible. That said, it sounds like we may need to include a Chapter just for METplus at some point. 
@mark-a-potts Any decision about including METplus in SRW release, or should we put any changes regarding METplus on hold for now?